### PR TITLE
[LPT] Unexpected quantisation level fix

### PR DESCRIPTION
**Pull request by eshoguli**
### Details:
 - *[LPT] Unexpected quantisation level fix*

### Tickets:
 - *CVS-112304*

### CODE REVIEW

File: src/plugins/intel_cpu/tests/functional/shared_tests_instances/low_precision_transformations/convolution_transformation.cpp [hunk 1]
```diff
@@ -116,6 +116,34 @@ const std::vector<LayerTestsDefinitions::ConvolutionTransformationParam> params
...
+    // TODO: US #116899
+    // not supported quantization level on weights
+    //{
+    //    { 256ul, ngraph::Shape { 1, 1, 1, 1 }, { 0.f }, { 25.5f }, { 0.f }, { 25.5f } },
+    //    false,
+    //    { 65536ul, ngraph::Shape{1, 1, 1, 1}, {0.f}, {254.f}, {-12.7f}, {12.7f}},
+    //    false,
+    //    "Convolution",
+    //    "FP32"
+    //},
```
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/plugins/intel_cpu/tests/functional/shared_tests_instances/low_precision_transformations/convolution_transformation.cpp:146]: I'm going to implement this feature for ALL operations in US #116899.

File: src/plugins/intel_gpu/tests/functional/shared_tests_instances/low_precision_transformations/convolution_transformation.cpp [hunk 2]
```diff
@@ -84,6 +84,34 @@ const std::vector<LayerTestsDefinitions::ConvolutionTransformationParam> params
...
+    // TODO: US #116899
+    // not supported quantization level on weights
+    //{
+    //    { 256ul, ngraph::Shape { 1, 1, 1, 1 }, { 0.f }, { 255.f }, { 0.f }, { 25.5f } },
+    //    false,
+    //    { 65536ul, ngraph::Shape{1, 1, 1, 1}, {0.f}, {254.f}, {-12.7f}, {12.7f}},
+    //    false,
+    //    "Convolution",
+    //    "FP32"
+    //},
```
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/plugins/intel_gpu/tests/functional/shared_tests_instances/low_precision_transformations/convolution_transformation.cpp]: done

File: src/plugins/intel_cpu/tests/functional/shared_tests_instances/low_precision_transformations/convolution_transformation.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/plugins/intel_cpu/tests/functional/shared_tests_instances/low_precision_transformations/convolution_transformation.cpp]: done

**Review by v-Golubev — APPROVED** [review]
In general LGTM

**Pull request merged**